Gemfile.jruby-2.3.lock.release

```diff
@@ -312,12 +310,10 @@ GEM
       logstash-codec-plain
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       stud
-    logstash-input-http (3.0.10)
+    logstash-input-http (3.2.0-java)
```

Comment:
@jsvd - do you feel comfortable enough with the passivity of this to upgrade in a patch release ?

Reply:
let's skip this